src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -299,6 +299,7 @@ def reinit_data(self):
         self.db_conn.connect(mibs.APPL_DB)
         mgmt_ip_bytes = self.db_conn.get(mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE, b'lldp_loc_man_addr')
 
+        self.man_addr_list = []
```

Comment:
Let's try to follow master branch as much as possible to prevent future conflict if we later decide cherry-pick something. Move this line to the beginning of this function, after the heredoc, and add one more empty line after.

Reply:
Fixed.